### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by hemajv**
## Related Issues and Dependencies

#5 

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

Adding more description to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+
+The repository is organized into the following directories:
+
+* [`data-presentation`](https://github.com/operate-first/data-driven-development/tree/main/data-presentation) - This folder comprises of documentation/guides and best practices on how data can be represented in different ways by using various tools such as creating dashboards in [Grafana](https://grafana.com/) or [Superset](https://superset.apache.org/), developing data visualizations using [Jupyter notebooks](https://jupyter.org/) etc.
+
+* [`data-sources`](https://github.com/operate-first/data-driven-development/tree/main/data-sources) - This folder comprises of documentation and guides for various data sources used in the team such as [Prometheus](https://prometheus.io/), GitHub etc, what type of data is collected from these sources and in what context these data sources can be used.
+
+* [`notebooks`](https://github.com/operate-first/data-driven-development/tree/main/notebooks) - This folder comprises of Jupyter notebooks providing examples on how to interact with the different data sources and perform analysis on top of the data.
 
 ## Template Features
```
Comment (oindrillac) [inline README.md:21]: Is it safe to remove this template section?

**Review by oindrillac — COMMENTED** [review]
/lgtm 👍

File: README.md (hunk 1 above)
Comment (hemajv) [reply README.md:21]: ah yeah I think we can remove it

**Review by oindrillac — APPROVED** [review]
👍

**Pull request merged**